src/main.cpp

```diff
@@ -8538,6 +8538,83 @@ int main_test(int argc, char** argv){
     return vg::unittest::run_unit_tests(argc, argv);
 }
 
+void help_sort(char** argv){
+    cerr << "usage: " << argv[0] << " sort [options] -i input_file -r reference_name > sorted.vg " << endl
```

Comment:
Writing help functions to stderr is bad practice. One of the few sane use case for this is when an invalid argument is passed through, e.g. 'vg sort --awwfdfsdvs'.